c-pallets/file-bank/src/lib.rs

```diff
@@ -332,7 +340,7 @@ pub mod pallet {
 				}
 				log::info!("End lease expiration check");
 			}
-			0
+			0 // Returned always 0 weight? Depending on the logic path there are some DB read/writes
```

Comment:
Thank you very much. I misunderstood the use of the initialize method before. I thought that only 0 would be returned here according to some examples I saw.
Now I have calculated the weight consumed for it:
https://github.com/CESSProject/cess/pull/88/files#diff-9c37b1a61a6e9394aac1e9383bf9d3fac9539378d1af8fcd75aed4f8a4f42967R297-R357